packages/ember/package.json

```
@@ -0,0 +1,88 @@
+{
+  "name": "@sentry/ember",
+  "version": "5.20.0",
+  "description": "Offical Sentry SDK for Ember.js",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/ember",
+  "author": "Sentry",
+  "license": "BSD-3-Clause",
+  "keywords": [
+    "ember-addon"
+  ],
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint:hbs": "ember-template-lint .",
+    "lint:js": "eslint .",
+    "start": "ember serve",
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each",
+    "prepublishOnly": "ember ts:precompile",
+    "postpublish": "ember ts:clean"
+  },
+  "dependencies": {
+    "@sentry/browser": "^5.20.0",
+    "@sentry/types": "^5.20.0",
+    "@sentry/utils": "^5.20.0",
```

```suggestion
    "@sentry/browser": "5.20.0",
    "@sentry/types": "5.20.0",
    "@sentry/utils": "5.20.0",
```

These should be fully version locked, otherwise lerna has troubles.